.github/workflows/argus-docker-build.yaml

```diff
@@ -127,7 +127,9 @@ jobs:
           platform: ${{ matrix.image.platform }}
           build_args: ${{ matrix.image.build_args }}
           secret_files: ${{ matrix.image.secret_files }}
-          image_tag: ${{ needs.prep.outputs.image_tag }}
+          image_tag: |
+            ${{ needs.prep.outputs.image_tag }}
+            ${{ (contains(inputs.envs, 'prod') && 'production') || '' }}
```

Comment:
Will this be a floating tag? I thought you couldn't have the same tag on multiple images